zrak/zrak/Builders/BlogBuilder.cs

```diff
@@ -22,5 +23,15 @@ public BlogModel Build(BlogStoreModel blogStoreModel)
                 Body = blogStoreModel.Body
             };
         }
+
+        public BlogStoreModel BuildId(BlogModel blogModel)
+        {
+            return new BlogStoreModel

```

Comment:
You should combine with another build function and have an if

Reply:
Added in Delete Post